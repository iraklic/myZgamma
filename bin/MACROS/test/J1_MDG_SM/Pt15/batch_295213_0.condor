000 (295213.000.000) 11/07 10:09:49 Job submitted from host: <131.225.191.208:47381>
...
001 (295213.000.000) 11/07 10:10:43 Job executing on host: <131.225.206.188:50714>
...
006 (295213.000.000) 11/07 10:11:53 Image size of job updated: 82136
...
005 (295213.000.000) 11/08 02:18:13 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 16:05:49, Sys 0 00:00:25  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 16:05:49, Sys 0 00:00:25  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	127179  -  Run Bytes Sent By Job
	21846248  -  Run Bytes Received By Job
	127179  -  Total Bytes Sent By Job
	21846248  -  Total Bytes Received By Job
...
