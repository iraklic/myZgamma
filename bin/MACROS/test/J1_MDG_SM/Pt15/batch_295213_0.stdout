
CMSSW on Condor

started at Mon Nov  7 10:10:45 CST 2011

parameter set:
CONDOR_CLUSTER: 295213
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu_B_S.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_295213_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_295213_0.log 2>&1\n

finished at Tue Nov  8 02:18:13 CST 2011
